pulsar-client/src/main/java/org/apache/pulsar/client/util/NameUtil.java

```diff
@@ -21,7 +21,7 @@
 import java.util.UUID;
 import org.apache.commons.codec.digest.DigestUtils;
 
-public class ConsumerName {
+public class NameUtil {
     public static String generateRandomName() {
         return DigestUtils.sha1Hex(UUID.randomUUID().toString()).substring(0, 5);
     }
```

Comment:
why was this class renamed? This is not a generic method that we'd like to reuse more broadly. The original method `generateRandomName` is not even describing the name properly.

Reply:
Now I see the reason of renaming the class, https://github.com/apache/pulsar/pull/23577#discussion_r1846456815 .
I'd suggest removing this class completely in a follow up PR and using Commons Lang3's `org.apache.commons.lang3.RandomStringUtils#randomAscii(int)` method with the parameter `5` to create a random string of 5 characters. @geniusjoe Would you like to submit a PR for that?

Reply:
> Now I see the reason of renaming the class, [#23577 (comment)](https://github.com/apache/pulsar/pull/23577#discussion_r1846456815) . I'd suggest removing this class completely in a follow up PR and using Commons Lang3's `org.apache.commons.lang3.RandomStringUtils#randomAscii(int)` method with the parameter `5` to create a random string of 5 characters. @geniusjoe Would you like to submit a PR for that?

@lhotari  Thank you for reviewing my pr. This `generateRandomName()` only use in just few cases and I think it's better to use `randomAscii()` you mentioned to replace this class. I'm glad to submit a new PR ASAP.

Reply:
@lhotari 
Hello Lari I find that `randomAscii(int)` will return a string which also contains symbols and signs, example below:
```
~ # for ((i=32;i<127;i++)) do printf "\\$(printf %03o "$i")"; done;printf "\n"                      root@VM-152-173-centos
 !"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmnopqrstuvwxyz{|}~
```
The original `generateRandomName()` function uses lower-case UUID substring (i.e. uses 0-9 and a-f), so that it might be a break changes when we replace this function with `randomAscii(int)`. Shall we choose another random string generator function?

Reply:
@geniusjoe good point. instead of `randomAscii`, we could use `randomAlphanumeric` in `RandomStringUtils`.